geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreTest.scala

```diff
@@ -48,12 +48,13 @@ class HBaseDataStoreTest extends Specification with LazyLogging {
     "work with points" in {
       val typeName = "testpoints"
 
+      System.setProperty("geomesa.hbase.remote.filtering", "true")
```

Comment:
Actually, since I removed the ability to run without remote, I need to get rid of this.  But see the previous comment.